css/main.css

```diff
@@ -20,7 +20,7 @@ body {
 nav {
   width: 100%;
   text-align: right;
-  margin-top: 10px;
+  /* margin-top: 10px; */
```

Comment:
It's a good practice not to keep commented code in our production code.

Reply:
Okay will keep this in mind from next  time.

Reply:
Changes Have Been Done. Please Review And Merge.